### PR TITLE
tests: addition of test_examples_app.py

### LINKED ISSUE #21: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:

- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
* Adds test_examples_app.py. (closes #21)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: tests/test_examples_app.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Test example app."""
+
+import datetime
```
Comment (jirikuncar) [inline tests/test_examples_app.py]: unused

File: tests/test_examples_app.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    os.killpg(webapp.pid, signal.SIGTERM)
+    # return to the original directory
+    os.chdir(current_dir)
+
+
+def test_example_app(example_app):
+    """Test example app."""
+    # Testing get index page
+    cmd = 'curl http://127.0.0.1:5000/'
+    output = subprocess.check_output(cmd, shell=True).decode("utf-8")
```
Comment (jirikuncar) [inline tests/test_examples_app.py]: `output` is not used

File: tests/test_examples_app.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    assert 'Hello world!'
```
Comment (jirikuncar) [inline tests/test_examples_app.py]: ?

**Comment by jirikuncar** [comment]
@hachreak please add yourself to `AUTHORS.rst`

File: tests/test_examples_app.py (hunk 1 above)
Comment (hachreak) [inline tests/test_examples_app.py]: done

File: tests/test_examples_app.py (hunk 2 above)
Comment (hachreak) [inline tests/test_examples_app.py]: done

File: tests/test_examples_app.py (hunk 3 above)
Comment (hachreak) [inline tests/test_examples_app.py]: fixed XD

**Comment by hachreak** [comment]
added!
:shipit: ? :)

**Pull request merged**